src/shared/config/config.go

```diff
@@ -11,24 +11,35 @@ Shared config keys between all reporter components - DNS Sniffer, Kafka watcher,
 */
 
 const (
-	MapperApiUrlKey              = "mapper-api-url"
-	MapperApiUrlDefault          = "http://mapper:9090/query"
-	DebugKey                     = "debug"
-	DebugDefault                 = false
+	MapperApiUrlKey     = "mapper-api-url"
+	MapperApiUrlDefault = "http://mapper:9090/query"
+
+	DebugKey     = "debug"
+	DebugDefault = false
+
 	PrometheusMetricsPortKey     = "metrics-port"
 	PrometheusMetricsPortDefault = 2112
+
 	TelemetryErrorsAPIKeyKey     = "telemetry-errors-api-key"
 	TelemetryErrorsAPIKeyDefault = "d86195588a41fa03aa6711993bb1c765"
-	EnableTCPKey                 = "enable-tcp"
-	EnableTCPSnifferDefault      = true
-	EnableSocketScannerKey       = "enable-socket-scanner"
-	EnableSocketScannerDefault   = true
-	EnableDNSKey                 = "enable-dns"
-	EnableDNSSnifferDefault      = true
+
+	EnableTCPKey            = "enable-tcp"
+	EnableTCPSnifferDefault = true
+
+	EnableSocketScannerKey     = "enable-socket-scanner"
+	EnableSocketScannerDefault = true
+
+	EnableDNSKey            = "enable-dns"
+	EnableDNSSnifferDefault = true
 
 	EnvPodKey       = "pod"
 	EnvNamespaceKey = "namespace"
 
+	// experimental features
+
+	EnableEBPFKey     = "experimental-ebpf"
```

Comment:
should be called `enable-ebpf` (the experimental portion can go into description), or `enable-ebpf-experimental` if you really insist on including the experimental part in the flag name.